src/main/anovos/data_transformer/transformers.py

```diff
@@ -423,6 +424,84 @@ def monotonic_binning(
     return odf
 
 
+def cat_to_num_transformer(
```

Comment:
this method should support the supervised method type too. We introduce two additional parameters such as method_type={"supervised" or "unsupervised"}
encoding="one-hot" or "label". We need to have the right error handling technique to avoid the wrong combination of inputs

Reply:
checked in the changes based on the discussion